src/HandlesStoredEvent.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+
+namespace Spatie\EventProjector;
```

Comment:
I don't see a reason why this trait exists, put the functionality back in the main class.